kclvm/tools/src/LSP/src/main_loop.rs

```diff
@@ -11,6 +13,9 @@ pub(crate) fn main_loop(
     config: Config,
     initialize_params: InitializeParams,
 ) -> anyhow::Result<()> {
+    let kcl_watch_system = KCLWatchSystem::new(conf.clone());
+    kcl_watch_system.start_observer(); // Start the observer here
```

Comment:
Should it serve on PWD or in the user's workspace?

Reply:
currently for PWD

Reply:
Why for pwd. In pwd how we watch user kcl files.

Reply:
files will be watched based on the path user will defined in the config file, and the watcher will watch the files based on the file extension and trigger the corresponding event handler.